requestlog/requestlog.go

```diff
@@ -107,6 +112,7 @@ type Entry struct {
 	ResponseHeaderSize int64
 	ResponseBodySize   int64
 	Latency            time.Duration
+	TraceID            string
 }
```

Comment:
We also should capture the [span ID](https://opencensus.io/tracing/span/). (Similar to the above comment, let's use the `trace.SpanID` type for this field.)